src/scss/custom.scss

```diff
@@ -15,6 +39,9 @@
   // --ifm-color-primary-lighter: rgb(102, 212, 189);
   // --ifm-color-primary-lightest: rgb(146, 224, 208);
   // --ifm-code-font-size: 95%;
+  --ifm-font-size-base: 0.9em;
```

Comment:
why did you use `0.9em`? what about using `1rem` instead?

Reply:
@deadpine I only had the reference site to pull from so used `em` from there, can use `rem` if prefered